src/Constants.h

```diff
@@ -95,6 +96,16 @@ constexpr const char* ARDUPILOT_PROTOCOL_NAME = "/ardupilot";
 constexpr std::chrono::milliseconds JOINT_POWER_REPEAT_PERIOD(333);
 constexpr std::chrono::milliseconds ARM_IK_UPDATE_PERIOD(50);
 
+constexpr std::array<robot::types::motorid_t, 4> WHEEL_IDS = {
+	robot::types::motorid_t::frontLeftWheel,
+	robot::types::motorid_t::frontRightWheel,
+	robot::types::motorid_t::rearLeftWheel,
+	robot::types::motorid_t::rearRightWheel,
+};
+
+const extern std::unordered_map<DriveMode, std::array<int32_t, 4>> WHEEL_ROTS;
+constexpr double STEER_EPSILON = 1000;
```

Comment:
Can we put this in a nested namespace, so maybe it would be `Constants::Drive` or something?